### DISCUSSION TITLE
Authentication with firebase

### DISCUSSION THREAD

**Discussion by rahmanprofile** [rrousselGit/provider#775] [Q&A]
**Describe the bug**
Hi 
 i am  using provider but there is a big problem in authentication when we setup our flutter app its giving me lots of error and the most important things after fixing the issue it does not fix and giving lots of issue tottaly i am dissatishfied of your provider because it's takes my lot's of time ....?

Thank you Developer Team (_'_)

**Comment by bensonarafat** [comment]
Hi
Can you show the errors you are having 
Glad to help

**Comment by rahmanprofile** [comment]
Sure , i have logged but there's one method today which i have used that
can send ...?

**Comment by rahmanprofile** [comment]
On Tue, 23 Aug 2022 at 10:41, Obaidur Rahman ***@***.***>
wrote:

> Sure , i have logged but there's one method today which i have used that
> can send ...?
>
> On Tue, 23 Aug 2022, 10:40 am Benson Arafat, ***@***.***>
> wrote:
>
>> Hi
>> Can you show the errors you are having
>> Glad to help
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/rrousselGit/provider/issues/772#issuecomment-1223551970>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AZYFWH5DECUTNUAXYSOGSADV2RMKTANCNFSM57J7LI7Q>
>> .
>> You are receiving this because you authored the thread.
>>
>
Thank you for understanding me is there have any issue in my authentication
process then sir any method that is best you know please provide me source
code .

**Comment by bensonarafat** [comment]
Have you provided your error log and some code?

**Comment by rahmanprofile** [comment]
Yes

**Comment by bensonarafat** [comment]
I don't think you have.
